app/src/main/java/com/eventyay/organizer/core/attendee/list/SwipeController.java

```diff
@@ -0,0 +1,100 @@
+package com.eventyay.organizer.core.attendee.list;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.helper.ItemTouchHelper;
+import android.view.View;
+
+import com.eventyay.organizer.R;
+
+public class SwipeController extends ItemTouchHelper.SimpleCallback {
+
+    private final AttendeesPresenter attendeesPresenter;
+    private final Paint paintGreen = new Paint();
+    private final Paint paintRed = new Paint();
+    private final Bitmap closeIcon;
+    private final Bitmap doneIcon;
+
+    public SwipeController(AttendeesPresenter attendeesPresenter, Context context) {
+        super(0, ItemTouchHelper.RIGHT | ItemTouchHelper.LEFT);
+        this.attendeesPresenter = attendeesPresenter;
+
+        closeIcon = BitmapFactory.decodeResource(context.getResources(), R.drawable.close);
+        doneIcon = BitmapFactory.decodeResource(context.getResources(), R.drawable.done);
+
+        paintGreen.setColor(context.getResources().getColor(R.color.light_green_500));
+        paintRed.setColor(context.getResources().getColor(R.color.red_500));
+    }
+
+    @Override
+    public boolean onMove(RecyclerView recyclerView, RecyclerView.ViewHolder viewHolder, RecyclerView.ViewHolder target) {
+        return false;
+    }
+
```

Comment:
Add `MovementFlags` so that swiping left will check-out the attendee and swiping right will check-in. Based on the checked-in/out status only one direction of swipe should be enabled at a time.

Reply:
Okay. I thought that this might confuse the user so I did not implement it in this way.

Reply:
Updated. Added new GIF also.

Reply:
This is the standard way also followed by other apps.

> Okay. I thought that this might confuse the user so I did not implement it in this way.